app/log-trace-message.js

```diff
@@ -0,0 +1,7 @@
+const appInsights = require('applicationinsights')
```

Comment:
You probably need to change the name to something like 'app-insights-....' as in the future we may want to track excpetions, events, etc.

Reply:
Agreed, see other comment